lib/commons/dom/is-visible.js

```diff
@@ -208,6 +209,15 @@ function isVisible(el, screenReader, recursed) {
     return false;
   }
 
+  if (nodeName === 'summary') {
+    return true;
+  }
+
+  // Handle lack of support in IE
+  if (nodeName === 'details' && 'open' in el && !el.hasAttribute('open')) {
```

Comment:
I'm not sure what to suggest here, but it jumped out at me that if `el` is somehow not an object, this is going to throw. That doesn't seem possible in this function as written today, but that's the kind of thing that could slip through in a refactor. I fee like a little defensive programming here could save us from a headache in the future.